### ISSUE TITLE
Demo - Drops - not tab powered friendly

### ISSUE THREAD

**Issue by BilboAtBagEnd** [titon/toolkit#116]
Let me know if I should or shouldn't submit issues about the demo pages. 

Accessibility concern for the Drops demo: 

- Main Drops have no focus indication for keyboard navigation (tabbing)
- Nested drops do not activate on keyboard navigation to them nor on hitting enter to activate

**Comment by milesj** [comment]
I'd say most of the demos aren't tab friendly, so feel free to report anything. They are a work in progress.

**Comment by TraderStf** [comment]
If you allow me, what is often missing in frameworks is a sample site explaining the best way to use it, how to start.

A tiny project for newbies with do and don't to use it, to avoid him to 'hack' the framework, aware him of usual pitfalls (and perhaps tips and tricks). Small notes would be enough, just to highlight them to newcomers.

The demo, the testing pages and this tutorial project could be joined together.

Just an idea, don't know if it is realistic, feasible, and it is perhaps useless for the intended audience, other less techies will head to a CMS.

**Comment by milesj** [comment]
I updated the main nav with tab support. The components themselves will need to be updated in 3.0 to support full tabbed browsing.
